clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1525,17 +1525,22 @@ def create_file(
     output_path.mkdir(parents=True, exist_ok=True)
 
     # If updated mode is selected, check if an old image is existing and remove it
-    for image_to_remove in output_path.glob(f"{output_filename}*"):
+    rgx_existing = re.compile(
```

Comment:
Thanks for the suggestion @NicolasGensollen ! In fact I was thinking of moving all the blocks of logic for verification/checking from `create_files` in helpers functions when I was working on it yesterday. I struggle with finding the appropriate way to get out of the `create_files` when necessary though. I tried something to handle existing images as you said but it might be a bit redundant with the `mod_to_update` parameter,

WDYT ?